sdks/python/tests/unit/api_objects/attachment/test_converters.py

```diff
@@ -59,7 +59,7 @@ def test_attachment_to_message__file_name():
         entity_type="trace",
         entity_id=entity_id,
         project_name=project_name,
-        base_url=base_url,
+        url_override=url_override,
```

Comment:
let's maybe rename it to `encoded_url_override` to make it less confusing.

Reply:
Renamed where appropriate